packages/docs/src/lang/en/customization/SassVariables.json

```diff
@@ -10,5 +10,7 @@
   "sassAlert2": "If you are requiring the Vuetify stylesheet by a `<link>` tag in your index file, remove it.",
   "sassText5": "Under the hood, the new variables will be hoisted to the top of the compiled styles and used to override the default behavior.",
   "sfcHeader": "## Single-file components",
-  "sfcText1": "When vue-cli compiles styles, by default it does not associate different file types for SASS, (**.scss** and **.sass**). When adding custom styles in a SFC (single-file component) with `lang=\"sass\"`, it will throw an error saying that a _semi-colon_ must be added to the import statement. When you add this _semi-colon_, it will tell you to remove it. This is due to the fact that Vuetify uses both **.scss** and **.sass** files. In order to resolve this, we have to tell webpack to explicitly use the **sass-loader** for **.sass** files."
+  "sfcText1": "Using the above configuration you will also be able to access Vuetify variables in SFC files when using style blocks with `lang=\"sass\"`. However if you wish to use `lang=\"scss\"`, an extra step is necessary.",
```

Comment:
Don't we have scss files too?

Reply:
Isnt that what its trying to say is that the config will work if using `sass` and if you want to uses `scss`, there some extra work involved? `However if you wish to use lang=\"scss\", an extra step is necessary."` ?

Reply:
We have scss files, which should be affected by that. @nekosaur did you test this with a bunch of components actually loaded, or just an empty project?

Reply:
I tested by using the default template that comes with vue-cli vuetify plugin, and added a few random components. Things seemed to work just fine.